slick/src/main/scala/slick/sql/SqlCapabilities.scala

```diff
@@ -16,6 +16,8 @@ object SqlCapabilities {
   val sequenceMax = Capability("sql.sequenceMax")
   /** Supports min value for sequences */
   val sequenceMin = Capability("sql.sequenceMin")
+  /** supports interpreting strings as VARCHAR. Some databases eg. HSQL do not do this. */
+  val stringIsVarchar = Capability("sql.stringIsVarchar")
```

Comment:
I don't think it belongs in SqlCapabilities. If it's for testing only, put it into TestDB.capabilities. The distinction is handled internally in HsqldbDriver anyway, without looking at this flag.